README.md

```diff
@@ -2,28 +2,75 @@
 
 Merge Gatekeeper provides extra control for Pull Request management.
 
-## 🌄 What does Merge Gatekeeper provide, and Why?
+## 🌄 What Does Merge Gatekeeper Provide, and Why?
 
 <!-- == imptr: background / begin from: ./docs/details.md#[background] == -->
 
```

Comment:
this is completely better than before.
Originally it was clear enough on overall summary but unclear on what this actually does.
However, with the screenshot, what we really want to do and what Merge Gatekeeper does are very clear.

Reply:
Sorry I'm not sure what you mean by this comment. Are you saying it's good with this change, or I'd need to add more stuff to this?

Reply:
just my feedback throughout the whole changes.
Previous one was not bad at all, but the changes on this pull made README etc much better.